src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -93,12 +93,11 @@
     <f:textbox/>
   </f:entry>
 
-    <f:optionalBlock title="${%Use custom workspace volume}" field="customWorkspaceVolumeEnabled" inline="true">
-      <f:entry title="${%Workspace Volume}" description="${%Volume to use for sharing the workspace}">
-        <f:dropdownDescriptorSelector field="workspaceVolume" oneEach="true" hasHeader="true" addCaption="${%Add Workspace Volume}"
-                                      deleteCaption="${%Delete Workspace Volume}" />
-      </f:entry>
-    </f:optionalBlock>
+  <f:entry title="${%Workspace Volume}" description="${%Volume to use for sharing the workspace}">
+    <f:dropdownDescriptorSelector field="workspaceVolume" oneEach="true" hasHeader="true" addCaption="${%Add Workspace Volume}"
```

Comment:
I have complained to GH that I cannot offer code suggestions on a multiline comment. Or I can, but they cannot be applied mechanically.
```suggestion
```